dev/TEAL.md

```diff
@@ -286,12 +305,17 @@ Asset fields include `AssetHolding` and `AssetParam` fields that are used in `as
 | `pop` | discard value X from stack |
 | `dup` | duplicate last value on stack |
 | `dup2` | duplicate two last values on stack: A, B -> A, B, A, B |
+| `dig` | duplicate value N from the top of stack |
```

Comment:
description is ambiguous, `N` refers to the position in the stack, not to a value, right? This opcode is used to access the element stack[N-1] right? Does it then push in on the stack?

Reply:
Yes, that's right.  I used terminology as we did for `dup` which does not say it "pushes" just that it duplicates. How about
"Pushes the Nth value from the top of the stack. `dig 0` is equivalent to `dup`"